maker-frontend/src/MakerApp.tsx

```diff
@@ -98,7 +98,20 @@ export default function App() {
     );
 
     const walletInfo = useLatestEvent<WalletInfo>(source, "wallet");
-    const priceInfo = useLatestEvent<PriceInfo>(source, "btcusd_quote");
+    const btcPriceInfo = useLatestEvent<PriceInfo>(source, "btcusd_quote");
+    const ethPriceInfo = useLatestEvent<PriceInfo>(source, "ethusd_quote");
+
+    let [priceInfo, setPriceInfo] = useState<PriceInfo | null>(null);
+
+    useEffect(() => {
+        if (symbol === Symbol.btcusd) {
+            setPriceInfo(btcPriceInfo);
+        } else if (symbol === Symbol.ethusd) {
+            setPriceInfo(ethPriceInfo);
+        } else {
+            setPriceInfo(null);
```

Comment:
Should we add an error log here? Could be hard to find out if no quote is shown and nothing is logged. I would not bother with an alert or so because we are the only one using this UI.